### PR TITLE
Change type of swagger file from JSON to YALM

### PR DESCRIPTION
**Pull request by eluyema**

### CODE REVIEW

File: gateway/package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "nodemon": "2.0.9"
   },
   "dependencies": {
+    "@types/yamljs": "^0.2.31",
```
Comment (yevhenii-moroziuk) [inline gateway/package.json]: Move this to devDependencies

File: gateway/src/api/middlewares/index.ts [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
-import { Project } from 'hypecrafter-shared/enums';
-import swaggerUI from 'swagger-ui-express';
-import { handleError } from './error-handler';
-import { logger } from './logger';
-import { authorization } from './authorization';
-import { Services } from '../../services';
-import openApiDocumentation from '../../../openApiDocumentation.json';
-import { env } from '../../env';
-import { WHITE_ROUTES } from '../../common/constants/whiteRouts';
+import express, { Express, json } from "express";
```
Comment (yevhenii-moroziuk) [inline gateway/src/api/middlewares/index.ts]: Do you have prettier? If so, change the config to use single quotes. Do you have eslint installed?

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: gateway/openApiDocumentation.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
+openapi: 3.0.0
+info:
+  title: API
+  description: API documentation for BSA-Project [Hypecrafter](https://github.com/BinaryStudioAcademy/bsa-2021-hypecrafter)
+  version: 0.1.9
+servers:
+  - url: http://localhost:3001/api
```
Comment (mtseluiko) [inline gateway/openApiDocumentation.yaml:7]: Could you also add the staging server URL?

**Review by yevhenii-moroziuk — APPROVED** [review]

**Comment by mtseluiko** [comment]
resolve conflicts, please

**Pull request merged**